extension-manifest-v3/scripts/build.js

```diff
@@ -197,3 +214,42 @@ for (const [id, path] of Object.entries(mapPaths(content_scripts))) {
     });
   }
 }
+
+if (argv.watch) {
+  buildPromise.then((watchEmitter) =>
+    watchEmitter.on('event', function callback(e) {
+      if (e.code === 'BUNDLE_END') {
+        watchEmitter.off('event', callback);
+
+        let settings;
+        switch (argv.target) {
+          case 'safari':
+            exec('xed xcode');
+            return;
+          case 'firefox':
+            settings = {
+              target: 'firefox-desktop',
+              firefoxBinary:
+                '/Applications/Firefox Nightly.app/Contents/MacOS/firefox-bin',
```

Comment:
can we make it platform independent?

Reply:
I thought about it, but it will be hard (would have to use some kind of `.env` file with paths), as there is no one place where you put your binaries. I assumed that this is mainly for us, which uses macOS, and we can keep browsers in predefined locations. I updated the README to highlight the problem. 

After all, the user can use `npm run build` and open the browser manually.

Reply:
maybe we can do something like this https://github.com/chrmod/puppeteer-webextension-test/blob/main/index.js#L25

Reply:
I think that web-ext has a way to tell it that you want to run nightly and it will look for it in commonly known path. With command line we can say `--firefox=nightly`

Reply:
I read a CLI help messages, even a source code, look for answers on google, stack... As far I can tell, if you want to use another version of Firefox, you must provide a path to the binary.